src/foam/dashboard/view/Card.js

```diff
@@ -7,7 +7,7 @@
 foam.CLASS({
   package: 'foam.dashboard.view',
   name: 'Card',
-  extends: 'foam.u2.View',
+  extends: 'foam.u2.Element',
```

Comment:
What's the reason for this change?

Reply:
This element is now being used as a border rather than a view